content/en/docs/install.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Installing Tekton Pipelines"
+linkTitle: "Install"
+weight: 20
+menu:
+  main:
+    weight: 20
+---
+
+To add Teketon to the Kubernetes Cluster on version 1.11 or later grant cluster-admin permissions to the current user.
+
+* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
```

Comment:
typo here `PIpelines` :stuck_out_tongue:

Reply:
lol, 😂 I fixed it.